torch/ao/quantization/utils.py

```diff
@@ -113,6 +113,15 @@ def weight_is_statically_quantized(qconfig):
     """
     return weight_dtype(qconfig) in [torch.quint8, torch.qint8]
 
+def op_is_int8_dynamically_quantized(qconfig) -> bool:
+    activation_dtype, weight_dtype, activation_compute_dtype = \
+        get_qconfig_dtypes(qconfig)
+    return (
+        activation_dtype is torch.float and
+        weight_dtype in (torch.quint8, torch.qint8) and
+        activation_compute_dtype in (torch.quint8, torch.qint8)
+    )
```

Comment:
assuming this is for fbgemm/qnnpack, should we explicitly list all possible combinations?
we only support activation_compute_dtype == torch.quint8 and weight_dtype == torch.qint8? or do we also support other combinations like (activation_compute_dtype == torch.qint8 and weight_dtype == torch.quint8)?

Reply:
sure, for now this is for fbgemm and qnnpack so I can update the code to reflect that, we can make it more general in a future PR